docs/source/developer/chapters.rst

```diff
@@ -11,17 +10,10 @@ contains the majority of educational material for the project.
   - :doc:`../author/chapters` - Provides detail on content in chapters application
   - :doc:`django_setup` - Provides detail of database structure
 
-In general the chapters application is a standard Django application, however it
-does store and update it's associated data for it's models uniquely.
+In general the chapters application is a standard Django application, however it does store and update it's associated data for it's models uniquely.
 
-Instead of creating and updating model objects through the website
-(for example: edited by an online editor),
-the objects are updated by a management script.
-The content for the chapters application is stored within the ``contents/``
-directory, as is run for each deployment of the system.
-Storing the content within the Git repository gives us greater control on
-reviewing and accepted proposed changes to content.
+Instead of creating and updating model objects through the website (for example: edited by an online editor), the objects are updated by a management script.
+The content for the chapters application is stored within the ``contents/`` directory, as is run for each deployment of the system.
+Storing the content within the Git repository gives us greater control over reviewing and accepting proposed changes to content.
 
-The management command for updating the applications data is ``loadchapters``
-(which is automatically called when running ``updatedata``), and can be found at
-``management/commands/loadchapters.py``.
+The management command for updating the applications data is ``loadchapters`` (which is automatically called when running ``updatedata``), and can be found at ``management/commands/loadchapters.py``.
```

Comment:
should updatedata be ./csfg update?

Reply:
update calls updatedata and updatedata calls loadchapters, so no.